### DISCUSSION TITLE
Ability to hide/disable specific glyphs

### DISCUSSION THREAD

**Discussion by kyledecot** [CoderLine/alphaTab#721] [Q&A]
Is it possible / are they plans to allow for hiding / disabling specific glyphs. For example in the screenshot below I would like to hide the "TAB" and bracket(?) glyphs.

<img width="1799" alt="Screen Shot 2022-01-22 at 1 48 56 PM" src="https://user-images.githubusercontent.com/192464/150651605-675b8066-4c52-42da-8754-d95ddd23a019.png">

**Comment by Danielku15** [comment]
Hi there. 
As of today there is not really any built in feature to achieve this. There might be some CSS hacks if you use SVG rendering but thats likely not sustainable. 

Lets talk a bit more about your actual requirements and goals. What exactly do you want to change on the visuals and why. This will help me understand your needs a lot better than the direct request to dybamically hide/shot some glypjhs. There are already settings like https://www.alphatab.net/docs/reference/settings/notation/elements/ and I could extend it based on new inputs.  ;)

**Reply by kyledecot** [reply]
So right now I'm only wanting to hide the two specific glyphs I mentioned above. First the first glyph, the website I'm building _only_ shows tablatures so it is redundant information to display to the user.  As for the second glyph I'm attempting to keep a minimal / clean UI so I'm wishing to remove elements that are redundant on each line.
